foursquare/__init__.py

```diff
@@ -954,7 +963,11 @@ def _process_response(response):
     # Default case, Got proper response
     if response.status_code == 200:
         return {"headers": response.headers, "data": data}
-    return _raise_error_from_response(data)
+    # Rate limit exceeded, user must wait for quota to reset in order to retrieve data
+    elif response.status_code == 403:
+        return {"headers": response.headers, "data": data}
```

Comment:
This seems a bit dangerous. The response structure looks the same as the 200.
We should probably raise an exception instead, with the `response.headers` included in it.